components/authz-service/storage/v2/conditionattribute.go

```diff
@@ -0,0 +1,76 @@
+package v2
```

Comment:
The name of this file seems odd. Should it be train-cased?

Reply:
Ah I thought that wasn't idiomatic in go but looks like it's ok on further googling. Can do.